test/test-util.tsx

```diff
@@ -1,22 +1,14 @@
 import React from 'react'
 import { render as rtlRender, RenderResult } from '@testing-library/react'
-import { CacheProvider, ThemeProvider } from '@emotion/react'
-import createCache from '@emotion/cache'
+import { ThemeProvider } from '@emotion/react'
 import { theme } from 'components/utils/styles/theme'
 import { ConfigProvider } from 'Context/Config'
 
 function render(ui: React.ReactElement, { ...options } = {}): RenderResult {
-  const emotionCache = createCache({
-    key: 'test-cache',
-  })
-  emotionCache.compat = true
```

Comment:
One concern with this is Emotion's errors about first-child return as they aren't right. But, it's better than the FOUC

Reply:
Ah in the test it doesn't matter actually. But it would appear in the console on the site itself.

Reply:
Ah yep, here we go:
![image](https://user-images.githubusercontent.com/9972287/171806617-d1f1005e-fefd-403b-9627-4df5b2eedcf4.png)

I'll try first-of-type and see what goes wrong

Reply:
Didn't seem to help anything, so I'm just going to merge as-is. Nothing breaks, and it resolves when JS loads

Looks like 5 rules affected, 3 in the agenda, and they're generally just about missing borders, e.g. the missing border next to highlighted (only affects the first group of 4)
![missing border](https://user-images.githubusercontent.com/9972287/175021157-fb22ab08-5745-4feb-b1cc-5a52768ad91d.png)